apps/nextra/components/IndexerBetaNotice.mdx

```diff
@@ -0,0 +1,5 @@
+import { Callout } from 'nextra/components'
+ 
+<Callout type="info">
+    The Indexer API, Transaction Stream Service, and Custom Processors are currently in beta. Please report any problems you encounter by creating an issue in the [aptos-indexer-processors](https://github.com/aptos-labs/aptos-indexer-processors/issues/new/choose) repo.
+</Callout>
```

Comment:
I think it may be better to export this as a `.tsx` custom component rather than `.mdx` so that the import path is consistent (`import { IndexerBetaNotice } from @components/index` rather than `import { IndexerBetaNotice } from @components/IndexerBetaNotice.mdx`). You'll need to create an `index.tsx` file that exports it. Lmk if you need help doing so